tensorrt_llm/commands/build.py

```diff
@@ -529,7 +564,8 @@ def main():
         'weight_only_precision': args.weight_only_precision,
     }
     parallel_build(source, build_config, args.output_dir, workers,
-                   args.log_level, model_cls, **kwargs)
+                   args.log_level, model_cls, args.dry_run,
+                   args.visualize_network, **kwargs)
```

Comment:
There is a `**kwargs` field, so we can reuse it to aovid too much changes on different functions.

```
 kwargs = {
        'logits_dtype': args.logits_dtype,
        'use_fused_mlp': args.use_fused_mlp,
        'weight_only_precision': args.weight_only_precision,
        'dry_run': args.dry_run,
        'visualize_network': args.visualize_network,
    }
```

Reply:
This is no longer necessary since we added `dry_run` and `visualize_network` to `BuildConfig` as suggested.